cenc-format.html

```diff
@@ -382,157 +374,92 @@ <h2 id="w3c-editor-s-draft-10-december-2015"><abbr title="World Wide Web Consort
   
     
       
         <p>
-          <em>This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current <abbr title="World Wide Web Consortium">W3C</abbr> publications and the latest revision of this technical report can be found in the <a href="http://www.w3.org/TR/"><abbr title="World Wide Web Consortium">W3C</abbr> technical reports index</a> at http://www.w3.org/TR/.</em>
+          This document is merely a <abbr title="World Wide Web Consortium">W3C</abbr>-internal  document. It
+          has no official standing of any kind and does not represent consensus of the <abbr title="World Wide Web Consortium">W3C</abbr>
+          Membership.
         </p>
         
-          
-          
       <p>The working group maintains <a href="https://github.com/w3c/encrypted-media/issues">a list of all bug reports that the editors have not yet tried to address</a>.</p>
       <p>Implementors should be aware that this specification is not stable. <strong>Implementors who are not taking part in the discussions are likely to find the specification changing out from under them in incompatible ways.</strong> Vendors interested in implementing this specification before it eventually reaches the Candidate Recommendation stage should join the mailing list mentioned below and take part in the discussions.</p>
     
-          
-          <p>
-            This document was published by the <a href="http://www.w3.org/html/wg/">HTML Working Group</a> as an Editor's Draft.
-            
-            
-              If you wish to make comments regarding this document, please send them to
-              <a href="mailto:public-html-media@w3.org">public-html-media@w3.org</a>
-              (<a href="mailto:public-html-media-request@w3.org?subject=subscribe">subscribe</a>,
-              <a href="http://lists.w3.org/Archives/Public/public-html-media/">archives</a>).
-            
-            
-            
-            
-            
-              
-              All comments are welcome.
-              
-            
-          </p>
-          
-          
-          
-            <p>
-              Publication as an Editor's Draft does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr>
-              Membership. This is a draft document and may be updated, replaced or obsoleted by other
-              documents at any time. It is inappropriate to cite this document as other than work in
-              progress.
-            </p>
-          
-          
-          
-          <p>
-            
-              This document was produced by
-              
-              a group
-               operating under the
-              <a id="sotd_patent" property="w3p:patentRules" href="http://www.w3.org/Consortium/Patent-Policy-20040205/">5 February 2004 <abbr title="World Wide Web Consortium">W3C</abbr> Patent
```

Comment:
The use of specStatus: "base" in ReSpec means there is no patent policy for the document. Is that correct/desirable?